bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/ReolinkHandler.java

```diff
@@ -0,0 +1,280 @@
+                        ipCameraHandler.setChannelState(CHANNEL_CAR_ALARM, OnOffType.ON);
+                    } else {
+                        ipCameraHandler.setChannelState(CHANNEL_CAR_ALARM, OnOffType.OFF);
+                    }
+                    break;
+                case "/api.cgi?cmd=GetAudioAlarmV20":
+                    if (content.contains("\"enable\" : 1")) {
+                        ipCameraHandler.setChannelState(CHANNEL_ENABLE_AUDIO_ALARM, OnOffType.ON);
+                    } else {
+                        ipCameraHandler.setChannelState(CHANNEL_ENABLE_AUDIO_ALARM, OnOffType.OFF);
+                    }
+                    break;
+                case "/api.cgi?cmd=GetIrLights":
+                    if (content.contains("\"state\" : 0")) {
+                        ipCameraHandler.setChannelState(CHANNEL_AUTO_LED, OnOffType.OFF);
+                    } else {
+                        ipCameraHandler.setChannelState(CHANNEL_AUTO_LED, OnOffType.ON);
+                    }
+                    break;
+                case "/api.cgi?cmd=GetMdState":
+                    if (content.contains("\"state\" : 0")) {
+                        ipCameraHandler.setChannelState(CHANNEL_MOTION_ALARM, OnOffType.OFF);
+                    } else {
+                        ipCameraHandler.setChannelState(CHANNEL_MOTION_ALARM, OnOffType.ON);
+                    }
+                    break;
+            }
+        } finally {
+            ReferenceCountUtil.release(msg);
+        }
+    }
+
+    // This handles the commands that come from the openHAB event bus.
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        if (command instanceof RefreshType) {
+            switch (channelUID.getId()) {
+                case CHANNEL_ENABLE_MOTION_ALARM:
+                    ipCameraHandler.sendHttpPOST("/api.cgi?cmd=GetMdState" + ipCameraHandler.reolinkAuth);
+                    break;
+                case CHANNEL_ENABLE_AUDIO_ALARM:
+                    ipCameraHandler.sendHttpPOST("/api.cgi?cmd=GetAudioAlarmV20" + ipCameraHandler.reolinkAuth,
+                            "[{ \"cmd\":\"GetAudioAlarmV20\", \"action\":1, \"param\":{ \"channel\": 0}}]");
+                    break;
+                case CHANNEL_AUTO_LED:
+                    ipCameraHandler.sendHttpPOST("/api.cgi?cmd=GetIrLights" + ipCameraHandler.reolinkAuth,
+                            "[{ \"cmd\":\"GetIrLights\"}]");
+                    break;
+            }
+            return;
+        } // end of "REFRESH"
+        switch (channelUID.getId()) {
+            case CHANNEL_ACTIVATE_ALARM_OUTPUT: // cameras built in siren
+                if (OnOffType.ON.equals(command)) {
+                    ipCameraHandler.sendHttpPOST("/api.cgi?cmd=AudioAlarmPlay" + ipCameraHandler.reolinkAuth,
+                            "[{\"cmd\": \"AudioAlarmPlay\", \"param\": {\"alarm_mode\": \"manul\", \"manual_switch\": 1, \"channel\": "
```

Comment:
Just checking, is "manul" correct, or should it be "manual"?

Reply:
It is correct, the Reolink API contains a number of spellings that look wrong.